docs/features/pouch_with_upgrade.md

```diff
@@ -0,0 +1,102 @@
+# Upgrade Container
+
+## Introduction
+
+Most containers within Alibaba are used in rich container mode. Some containers among the rich containers in traditional virtual machine O&M mode are still stateful. Updates and upgrades of stateful services are frequently performed daily operations within Alibaba. For the container technology that delivers images, the container operations corresponding to service update and upgrade include two steps: deletion of old image container and creation of the new image container.
```

Comment:
I do not think it is a good way to mention Alibaba in the general technical documents. Since what you invent is quite a general feature for the whole industry, explicit explanation for you feature is enough to make it clear.

Reply:
great advise